src/main/java/seedu/address/model/person/Organ.java

```diff
@@ -0,0 +1,57 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Person's organ in ORGANice.
+ * Guarantees: immutable; is valid as declared in {@link #isValidOrgan(String)}
+ */
+public class Organ {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Please key in a valid organ. Valid organ(s): kidney.";
+
+    public static final String VALIDATION_REGEX = "[^\\s].*";
+
+    public static final String KIDNEY = "kidney";
+
+    public final String value;
+
+    /**
+     * Constructs an {@code Organ}.
+     *
+     * @param organ A valid organ.
+     */
+    public Organ(String organ) {
+        requireNonNull(organ);
+        checkArgument(isValidOrgan(organ), MESSAGE_CONSTRAINTS);
+        value = organ;
+    }
+
+    /**
+     * Returns true if a given string is a valid organ.
+     */
+    public static boolean isValidOrgan(String test) {
+        return test.matches(VALIDATION_REGEX)
```

Comment:
i think `validation_regex` is redundant here because it basically say `accept any string` so put `test.equals` is enough i think

Reply:
Okay, removed it.